assets/src/edit-story/components/library/panes/text/textPane.js

```diff
@@ -57,17 +108,36 @@ function TextPane(props) {
       <Section
         title={__('Presets', 'web-stories')}
         titleTools={
-          <MainButton onClick={() => insertElement('text', DEFAULT_PRESET)}>
+          <MainButton
+            onClick={() => {
+              lastPreset.current = null;
+              insertElement('text', DEFAULT_PRESET);
+            }}
```

Comment:
Both of the `onClick` callbacks are getting fairly complex now - can we move them to variables defined above the return?